contracts/IrrevocableVesting.sol

```diff
@@ -40,9 +40,9 @@ contract IrrevocableVesting is IIrrevocableVesting {
         _;
     }
 
-    /// @dev This contract is means to be an implementation for
-    /// IrrevocableVestingFactory to clone. To prevent the implementation from
-    /// being used, the contract is rendered uninitializable.
+    /// @dev This contract is meant to serve as an implementation for
+    /// IrrevocableVestingFactory to clone. To prevent the implementation
+    /// from being used, the contract is rendered uninitializable.
```

Comment:
I also noticed this but it was too late and we already deployed some vestings so :man_shrugging: